src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -457,10 +459,14 @@ const WorkflowView = _.flow(
   async fetchInfo(savedConfig) {
     const { methodRepoMethod: { sourceRepo, methodNamespace, methodName, methodVersion, methodPath } } = savedConfig
     const { ajax: { Dockstore, Methods } } = this.props
+    const { methods } = this.state
     try {
       if (sourceRepo === 'agora') {
         const { synopsis, documentation, payload } = await Methods.method(methodNamespace, methodName, methodVersion).get()
-        this.setState({ synopsis, documentation, wdl: payload })
+        const snapshotIds = _.map(m => { return m.snapshotId },
```

Comment:
I'd specifically like feedback on improvements for how to get this array of snapshot Ids. I'm also not sure if this is the best place to be doing the transformation/saving it to state